src/auth/screens/events.screen.js

```diff
@@ -496,17 +503,43 @@ class Events extends Component {
     );
   }
 
+  renderFooter = () => {
```

Comment:
**YES**! This would be done in a subsequent PR and we'll have an issue to track FlatList to ApiPaginatedList migrations, like we did for styled-components